### PR TITLE
8 county data

### LINKED ISSUE #8: Counties Data Endpoint
## Description
Create an Endpoint that accepts a county as a path parameter and then returns that data set.

https://fastapi.tiangolo.com/tutorial/path-params/

REST Types: GET

Input Parameters: {county_name}

URL: /counties/{county_name}

## Acceptance Criteria
- [x] Returns: JSON object of that county's data
- [x] Add a sample JSON output to issue #5 for a front end developer to review.

### PR DESCRIPTION
**Pull request by jberns**
This closes #8 

This adds a JSON data output to the `/counties/{county}` endpoint.

If the `{county}` matches a value on the `counties_list` it will return a JSON response from the dataframe.

If the `{county}` does not match any of the values, it will return a 404 error.

### CODE REVIEW

**Comment by pjhoberman** [comment]
Awesome! I’ll do a full review when I get to a computer later today. Good
work.

File: backend/utils/data.py [hunk 1]
```diff
@@ -3,122 +3,116 @@
...
-  MHI = pd.read_csv("data/ACS-Median-Household-Income.tsv", sep='\t')
-  PB2 = pd.read_csv("data/ACS-Pop-below-200.tsv", sep='\t')
-  P = pd.read_csv("data/ACS-Population.tsv", sep='\t')
-  LEAPP = pd.read_csv("data/CDHS-Leap-Participants.tsv", sep='\t')
-  EAP = pd.read_csv("data/EOC-EA-Participants.tsv", sep='\t')
-  PES = pd.read_csv("data/EOC-Primary-Energy-Source.tsv", sep='\t')
-  UT = pd.read_csv("data/EOC-Utility-Territories.tsv", sep='\t')
-  XEAP = pd.read_csv("data/EOC-Xcel-EA-Participants.tsv", sep='\t')
+  AREA: str = area
+  processedData: pd.DataFrame = pd.DataFrame()
```
Comment (pjhoberman) [inline backend/utils/data.py:10]: The variable `processedData` doesn't end up getting used, as it gets set at line 115.

File: backend/utils/data.py [hunk 2]
```diff
@@ -3,122 +3,116 @@
 # %%
 import pandas as pd
 import numpy as np
```
Comment (pjhoberman) [inline backend/utils/data.py:5]: It doesn't look like numpy is used at all - might as well remove for now?

File: backend/utils/data.py (hunk 1 above)
Comment (jberns) [reply backend/utils/data.py]: Removed that as it is no longer needed.

File: backend/utils/data.py (hunk 2 above)
Comment (jberns) [reply backend/utils/data.py]: Removed

**Pull request merged**